dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -48,7 +48,7 @@ jenkins:
         location: "<%= agent['location'] %>"
         noOfParallelJobs: 1
         osDiskSize: <%= agent['osDiskSize'] ? agent['osDiskSize'] : @jcasc['agents_setup'][agent['os'].to_s]['osDiskSize'] %>
-        osDiskStorageAccountType: <%= agent['osDiskStorageAccountType'] ? agent['osDiskStorageAccountType'] : @jcasc['agents_setup'][agent['os'].to_s]['osDiskStorageAccountType'] %>
+        osDiskStorageAccountType: <%= agent['ephemeralOSDisk'] ? "Standard_LRS" : (agent['osDiskStorageAccountType'] ? agent['osDiskStorageAccountType'] : @jcasc['agents_setup'][agent['os'].to_s]['osDiskStorageAccountType']) %>
```

Comment:
IIUC the use of ephemeral disk implies this Standard_LRS type?

Reply:
Not from the Azure SDK. I had issue when testing in infra.ci.jenkins.io from the existing config: setting it to `Standard_LRS` is needed other wise it fails deploying the VM in Azure. => hence this code that ensures any user-custom config is overrident (but I did not found how to trigger a notice to puppet logs from the templates :'( )